content/beginner/090_rbac/map_iam_user_to_k8s_user.md

```diff
@@ -12,6 +12,8 @@ kubectl get configmap -n kube-system aws-auth -o yaml | grep -v "creationTimesta
 Next append the rbac-user mapping to the existing configMap
 
 ```
+export ACCOUNT_ID=$(aws sts get-caller-identity --output text --query Account)
```

Comment:
Normally all those scar are already set in the start of the workshop and added to .bashrc . Is it needed to read them here ?

Reply:
You're right, but I'm thinking more of those who want to attack than this part rather than the entire workshop. It's as you wish. Some "parts" already do "recalls" of env vars. This can also make sense in terms of consistency.